superbench/runner/ansible.py

```diff
@@ -33,14 +32,14 @@ def __init__(self, config):
             if inventory_list:
                 inventory_list = inventory_list.strip(',')
             if inventory_file or inventory_list:
-                self._config['inventory'] = inventory_file or inventory_list
                 self._config['host_pattern'] = 'all'
                 inventory = InventoryManager(loader=DataLoader(), sources=inventory_file or f'{inventory_list},')
                 host_list = inventory.get_groups_dict()['all']
                 if len(host_list) > 0:
                     self._config['cmdline'] = '--forks {}'.format(len(host_list))
                 if inventory_list in ['localhost', '127.0.0.1']:
                     self._config['cmdline'] += ' --connection local'
+                self._config['cmdline'] += ' --inventory {}'.format(inventory_file or f'{inventory_list},')
```

Comment:
Why add `,` at the end?

Reply:
if you do not add `, ` , it will be treated as a file name. Reference: https://stackoverflow.com/a/48155344/7630831